CHANGES.md

```diff
@@ -1,6 +1,27 @@
 Synapse 1.25.0rc1 (2021-01-06)
 ==============================
 
+Ending Support for Python 3.5 and Postgres 9.5
+----------------------------------------------
```

Comment:
Should this be above the rc1 release notes and get put into the 1.25.0 release notes?

Reply:
...yep. Fixed.